### PR TITLE
feat: persona based customisation for glossary and glossary term pages

### PR DESCRIPTION
**Pull request by chirag-madlani**
### Describe your changes:

Fixes: GEN-1441, GEN-1440, GEN-1439, GEN-1443, GEN-725
Partial fix: GEN-1438

https://github.com/user-attachments/assets/0379c338-ccf1-429c-93fb-b3c9da0cf19d

I worked on ... because ...

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/utils/AuthProvider.util.ts [hunk 1]
```diff
@@ -40,7 +40,7 @@ import { isDev } from './EnvironmentUtils';
 const cookieStorage = new CookieStorage();
 
 // 1 minutes for client auth approach
-export const EXPIRY_THRESHOLD_MILLES = 1 * 60 * 1000;
+export const EXPIRY_THRESHOLD_MILLES = 2 * 60 * 1000;
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/utils/AuthProvider.util.ts:43]: @chirag-madlani, I recommend making such changes in separate PR if required.

**Review by Sachin-chaurasiya — APPROVED** [review]
Left a few comments, thanks

File: openmetadata-ui/src/main/resources/ui/src/components/Glossary/GlossaryTerms/tabs/GlossaryOverviewTab.test.tsx [hunk 2]
```diff
@@ -48,22 +37,13 @@ jest.mock('../../../common/ResizablePanels/ResizablePanels', () => {
   ));
 });
 
-const onUpdate = jest.fn();
-
-describe('GlossaryOverviewTab', () => {
-  const selectedData = MOCKED_GLOSSARY_TERMS[0];
-  const permissions = MOCK_PERMISSIONS;
-
+describe.skip('GlossaryOverviewTab', () => {
```
[the remark this answers is not in the document]
Comment (chirag-madlani) [reply openmetadata-ui/src/main/resources/ui/src/components/Glossary/GlossaryTerms/tabs/GlossaryOverviewTab.test.tsx:40]: we were just checking few elements, will rewrite it with customisation

File: openmetadata-ui/src/main/resources/ui/src/components/MyData/CustomizableComponents/AddDetailsPageWidgetModal/AddDetailsPageWidgetModal.tsx [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+function AddDetailsPageWidgetModal({
+  open,
+  widgetsList,
+  addedWidgetsList,
+  handleCloseAddWidgetModal,
+  handleAddWidget,
+  maxGridSizeSupport,
+  placeholderWidgetKey,
+}: Readonly<Props>) {
+  const { t } = useTranslation();
```
[the remark this answers is not in the document]
Comment (chirag-madlani) [reply openmetadata-ui/src/main/resources/ui/src/components/MyData/CustomizableComponents/AddDetailsPageWidgetModal/AddDetailsPageWidgetModal.tsx:55]: Yes working on those separately

**Review by Sachin-chaurasiya — APPROVED** [review]

**Review by aniketkatkar97 — APPROVED** [review]

**Review by sonika-shah — APPROVED** [review]

**Pull request merged**